addons/binding/org.openhab.binding.amazondashbutton/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="amazondashbutton" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="dashbutton">
+		<label>@text/dashButtonLabel</label>
+		<description>@text/dashButtonDescription</description>
+
+		<channels>
+			<channel id="press" typeId="trigger-channel" />
```

Comment:
I changed it accordingly, but the label seems not to be resolved. I created an ESH issue:  [#2356](https://github.com/eclipse/smarthome/issues/2356)